src/fixtures/base.js

```diff
@@ -1,8 +1,8 @@
 // @ts-check
 import { test as base } from '@playwright/test';
-import { Application } from '../pages/Application';
+import Application from '../pages/Application';
```

Comment:
Why do we need these changes?
It would be better not to use the default export
Just disable this rule in the ESlint config